src/corefx/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -4,7 +4,12 @@ project(System.Security.Cryptography.Native)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 add_definitions(-DPIC=1)
-add_definitions(-DBIT64=1)
+
+if(CLR_CMAKE_PLATFORM_ARCH_AMD64)
+    add_definitions(-DBIT64=1)
+elseif(CLR_CMAKE_PLATFORM_ARCH_ARM)
+    add_definitions(-DBIT32=1)
+endif()
```

Comment:
going to stop repeating: "lets future proof arm64 with an else #error for now", ditto all the way down.